webapp/src/main/webapp/iaf/gui/views/txstorage/adapter_storage_list.html

```diff
@@ -30,42 +30,42 @@ <h4>Display Filters</h4>
 							<tbody>
 								<tr>
 									<th>Storage ID</th>
-									<td><input disabled icheck type="checkbox" ng-change="updateFilter('id')" ng-model="displayColumn.id"/></td>
+									<td><input disabled icheck type="checkbox" ng-change="updateFilter('id', storageType)" ng-model="displayColumn.id"/></td>
```

Comment:
I don't think you need to pass the storageType here, as it's already present as a `$scope.storageType` variable in the controller.